README.md

```diff
@@ -7,14 +7,13 @@ production quality app as quickly as possible.
 
 ## Getting Started 
 
-Before getting started, you may need some `Prerequisites`, to learn more you can check
-[Socket Runtime](https://sockets.sh/).
-**You'll also need to have Node>= 16 on your local developement machine.**
-That's it, you are ready to go!
+**You’ll need to have Node 16.0.0 or later version on your local development machine .**  
```

Comment:
we also can add this to the `package.json` file so `npm` will print a warning when the version of Node is less than 16
```json
  "engines": {
    "node": ">=16"
  }
```

Reply:
Just put this three lines on line 8 here https://github.com/socketsupply/create-socket-app/blob/main/package.json#L8